roles/ffh.mesh_wireguard/templates/vxlan.netdev.j2

```diff
@@ -0,0 +1,12 @@
+[NetDev]
+Name={{ vxlan_iface }}-{{ '%02d' % domain.id | int }}
+Kind=vxlan
```

Comment:
MACAddress=02:a1:71:04:{{ '%02d' % sn | int }}:{{ '%02d' % domain.id | int }}

Reply:
implemented in f6a88efb351275759aad1e2eeb6ada561a9d94d0